tests/test_citeprocpy.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+import pytest
+import unittest
+
+from api.citations.utils import render_citation
+
+
+class Node:
+    _id = '2nthu'
+    csl = {'publisher': 'Open Science Framework', 'author': [{'given': u'Henrique', 'family': u'Harman'}], 'URL': 'localhost:5000/2nthu', 'issued': {'date-parts': [[2016, 12, 6]]}, 'title': u'The study of chocolate in its many forms', 'type': 'webpage', 'id': u'2nthu'}
+
+
+class TestCiteprocpy:
+    @unittest.skip
```

Comment:
Please divide up the json into "tests you expect to pass" and "tests you expect to fail" and have two test methods, one to check each. If a tests you expect to pass fails or vice versa, report a failure, but I'd like these tests to run regardless.